File: ViewModels/PractitionersSearchFilterViewModel.cs
using KPProject.Models;
using System.Collections.Generic;

namespace KPProject.ViewModels
{
    public class PractitionersSearchFilterViewModel
    {
        public int StartingIndex { get; set; }
        public int EndingIndex { get; set; }
        public List<LanguageModel> LanguagesSelected { get; set; }
        public List<RegionModel> GeographicalLocationsSelected { get; set; }
        public Gender GenderSelected { get; set; }
    }
}

File: ClientApp/src/view-models/practitioners-search-filter-view-model.ts
import { GenderViewModel } from "./gender-view-model";
import { LanguageViewModel } from "./language-view-model";
import { RegionViewModel } from "./region-view-model";

export class PractitionersSearchFilterViewModel {
  public startingIndex: number;
  public endingIndex: number;
  public languagesSelected: Array<LanguageViewModel> = new Array<LanguageViewModel>();
  public geographicalLocationsSelected: Array<RegionViewModel> = new Array<RegionViewModel>();
  public genderSelected: GenderViewModel;
}
